web3/eth.py

```diff
@@ -359,6 +359,35 @@ def set_contract_factory(
     ) -> None:
         self.defaultContractFactory = contractFactory
 
+    def filter_munger(
```

Comment:
We can make this a static method since it doesn't use the `self` reference.
```python
    @staticmethod
    def filter_munger(
        filter_params: Optional[Union[str, FilterParams]] = None,
        filter_id: Optional[HexStr] = None,
    ) -> Union[List[FilterParams], List[HexStr], List[str]]:
```

Reply:
scratched